Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'ransack', '2.4.1'
 gem 'responders'
 gem 'sass', '~> 3.4.0' # this restriction originates from foundation-rails's version
 gem 'sass-rails', '< 5.1.0' # this restriction originates from the compass-rails's version
+gem 'webpacker', '~> 5'
```

Comment:
why ~> 5? Can we leave it open? we will have dependabot making upgrades and we will see if v6 will work or not.

Reply:
I think there's various breaking changes in `6.x`, it won't be a simple case of bumping the version.

Reply:
ok :+1: